release.nix

```
@@ -125,21 +134,26 @@ let
 
   jobs = {
     inherit dockerImageArtifact;
-    native =
+    linux =
```

`musl64` is also linux. Could it be like this?
```suggestion
    linux.native = ...;
    linux.musl64 = ...;
```